controller/workitem.go

```diff
@@ -256,6 +257,40 @@ func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
 	}
+	var space *space.Space
+	err = application.Transactional(c.db, func(appl application.Application) error {
+		// verify spaceID:
+		// To be removed once we have endpoint like - /api/space/{spaceID}/workitems
+		space, err = appl.Spaces().Load(ctx, ctx.SpaceID)
+		if err != nil {
+			log.Error(ctx, map[string]interface{}{
+				"space_id": ctx.SpaceID,
+			}, "unable to load space")
+			return errors.NewBadParameterError("space", "string").Expected("valid space ID")
+		}
+		return nil
+	})
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, err)
+	}
+
+	// A workaround for https://github.com/almighty/almighty-core/issues/1358
+	// Allow any user to create a work item in spaces belong to the "openshiftio" user
+	// Other spaces are open for the space collaborators only
+	// ----
+	spaceOwnerID := space.OwnerId.String()
+	// check both the "openshiftio" user and the "test" user from the test realm.
+	if "7b50ddb4-5e12-4031-bca7-3b88f92e2339" != spaceOwnerID && "ae68a343-c866-430c-b6ce-a36f0b38d8e5" != spaceOwnerID {
```

Comment:
please, let's create some constants for this in the same place, we created SystemSpace, etc...